<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(179, 179, 179);</string>
    </property>
    <widget class="QPushButton" name="pauseButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>110</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>pauza</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="organizmCounter">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_organizmy">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Ilość organizmów</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="timerLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>400</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(179, 179, 179);</string>
    </property>
    <widget class="QPushButton" name="continueButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>281</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Segoe UI&quot;;
background-color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>wznów</string>
     </property>
    </widget>
    <widget class="QPushButton" name="restartButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>281</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Segoe UI&quot;;
background-color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>restart</string>
     </property>
    </widget>
    <widget class="QPushButton" name="menu_exitButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>281</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Segoe UI&quot;;
background-color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>powrót</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_menu" native="true">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>60</y>
      <width>400</width>
      <height>401</height>
     </rect>
    </property>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>400</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="settingsButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>400</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>ustawienia</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>400</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>wyjście</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_settings" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(223, 223, 223);</string>
    </property>
    <widget class="QPushButton" name="menu_backButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>440</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QSlider" name="organizm_speed">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Prędkość poruszania się organizmów</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="org_speed">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>135</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ilość jedzenia potrzebna do rozmnożenia</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>75</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ilość jedzenia na mapie</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>801</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="food_slider1">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="food_slider2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="f_slider1">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>140</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="f_slider2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>80</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>801</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>801</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="s_label">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>202</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>s</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>190</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>powolne pojawianie się jedzenia</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="food_show_time">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>195</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="default_settings">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>440</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>domyślne
 ustrawienia</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="organism_time">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>195</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>181</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>długość życia organizmu</string>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>801</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>180</y>
       <width>16</width>
       <height>49</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="min_label">
     <property name="geometry">
      <rect>
       <x>357</x>
       <y>202</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>min</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>230</y>
       <width>16</width>
       <height>49</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>801</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="bad_organisms_number">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>244</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>181</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ilość złych organizmów</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>240</y>
       <width>291</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>pobrażenia zadawane przez zły organizm</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="bad_organism_damage">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>244</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>294</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Prędkość poruszania się złych organizmów</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bad_org_speed">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>300</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>801</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="bad_organizm_speed">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>300</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <zorder>menu_backButton</zorder>
    <zorder>organizm_speed</zorder>
    <zorder>label_2</zorder>
    <zorder>org_speed</zorder>
    <zorder>label</zorder>
    <zorder>label_3</zorder>
    <zorder>line</zorder>
    <zorder>food_slider1</zorder>
    <zorder>food_slider2</zorder>
    <zorder>f_slider1</zorder>
    <zorder>f_slider2</zorder>
    <zorder>line_4</zorder>
    <zorder>line_6</zorder>
    <zorder>s_label</zorder>
    <zorder>checkBox</zorder>
    <zorder>food_show_time</zorder>
    <zorder>default_settings</zorder>
    <zorder>organism_time</zorder>
    <zorder>label_4</zorder>
    <zorder>line_7</zorder>
    <zorder>line_2</zorder>
    <zorder>min_label</zorder>
    <zorder>line_8</zorder>
    <zorder>line_3</zorder>
    <zorder>bad_organisms_number</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>bad_organism_damage</zorder>
    <zorder>label_7</zorder>
    <zorder>bad_org_speed</zorder>
    <zorder>line_5</zorder>
    <zorder>bad_organizm_speed</zorder>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>590</y>
      <width>400</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(179, 179, 179);</string>
    </property>
    <widget class="QPushButton" name="restartButton_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>281</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Segoe UI&quot;;
background-color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>restart</string>
     </property>
    </widget>
    <widget class="QPushButton" name="menu_exitButton_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>281</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Segoe UI&quot;;
background-color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>menu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>22</pointsize>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Populacja nie przetrwała</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1308</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
